pyaedt/application/Variables.py

```diff
@@ -71,6 +71,7 @@ def unit_system(units):
     Parameters
     ----------
     units : str
+        Units for retrieving the associated unit system name.
     
     Returns
     -------
```

Comment:
Returns is only a bool when unsuccessful.  When successful this appears to return the key from `AEDT_Units`, which in this case is a `str`.  I would add an example like (e.g. `'AngularSpeed'`).